spark/src/test/scala/org/apache/spark/sql/delta/managedcommit/ManagedCommitTestUtils.scala

```diff
@@ -28,6 +29,18 @@ import org.apache.spark.sql.test.SharedSparkSession
 trait ManagedCommitTestUtils
   extends DeltaTestUtilsBase { self: SparkFunSuite with SharedSparkSession =>
 
+  def testWithoutManagedCommits(testName: String)(f: => Unit): Unit = {
```

Comment:
This is needed because `withSqlConf` doesn't support removing properties?
(maybe worth a doc comment)

Reply:
Yes - Adding doc comment in a followup PR.